### ISSUE TITLE
Fast and Accurate Network Embeddings via Very Sparse 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#54]
# TL;DR
For the task of dimensionality reduction of graph data while maintaining the distance between nodes, they proposed FastRP, which is 4000 times faster than DeepWalk, following the projective matrix initialization with Sparse Random Projection, by computing multi-step transitions iteratively. 

# Why it matters:

# Paper URL

https://arxiv.org/abs/1908.11512

# Submission Dates(yyyy/mm/dd)
2019/08/30

# Authors and institutions
Haochen Chen, Syed Fahad Sultan, Yingtao Tian, Muhao Chen, Steven Skiena
- Stony Brook University
- University of California 

# Methods

![image](https://user-images.githubusercontent.com/24524018/91037716-75fe8300-e644-11ea-9bad-d9fca2b677b4.png)

# Results
![image](https://user-images.githubusercontent.com/24524018/91037685-6848fd80-e644-11ea-845a-7721feb0e5d5.png)

# Comments
